src/qforte/ucc/spqe.py

```diff
@@ -439,16 +430,16 @@ def update_ansatz(self):
                     if(self._verbose):
                         print(f"  {Nmu_tup[1]:10}                  {np.real(Nmu_tup[0]):14}")
                     if(Nmu_tup[1] not in self._tops):
-                        self._tops.insert(0,Nmu_tup[1])
+                        self._tops.insert(0,self._excitation_dictionary[Nmu_tup[1]])
                         self._tamps.insert(0,0.0)
-                        self.add_from_basis_idx(Nmu_tup[1])
+                        self._nbody_counts[len(self._pool_obj[self._excitation_dictionary[Nmu_tup[1]]][1].terms()[0][1]) - 1] += 1
 
                 self._n_classical_params_lst.append(len(self._tops))
 
         else: # when M_omega == 'inf', proceed with standard SPQE
```

Comment:
What's `M_omega`? That's nto documented, either.

Reply:
M_omega is the total number of measurements for determining the residuals (see the M symbol in Eq. (16) of the original SPQE paper). Essentially, setting it to "inf" means perfect measurement of all residuals. I never played with anything else than "inf". I will add a comment with the explanation.